app/validation/validator.py

```diff
@@ -19,8 +19,12 @@ def __init__(self):
         with open('schemas/questionnaire_v1.json', encoding='utf8') as schema_data:
             self.schema = load(schema_data)
 
-    def validate_schema(self, json_to_validate):  # noqa: C901  pylint: disable=too-complex
-
+    def validate_schema(self, json_to_validate):
```

Comment:
How about: 
```suggestion
    def validate_schema(self, json_to_validate: str) -> List[Dict[str, str]]:
```

Reply:
I agree.... except a change like this I'd want to do over multiple functions in a single PR.
Would a raised issue/trello card be a fair compromise on it?

Reply:
Fair enough. I was really just testing suggested changes :)